test/fancylogger.py

```diff
@@ -30,16 +30,23 @@
 @author: Kenneth Hoste (Ghent University)
 @author: Stijn De Weirdt (Ghent University)
 """
+from __future__ import print_function
+
+import coloredlogs
 import logging
 import os
-from random import randint
 import re
 import sys
 import shutil
-
-from StringIO import StringIO
 import tempfile
+from random import randint
 from unittest import TestLoader, main, TestSuite
+
+try:
```

Comment:
use `is_py3` or move this to the py23 module

Reply:
done